pulumi/folio/deployments/R2-2021.yaml

```diff
@@ -1,70 +1,134 @@
 # Set this to 'true' the first time the deployment is done. Then make sure to set it to
```

Comment:
What is the difference between `R2-2021-modules.yaml` and `R2-2021.yaml`?

Reply:
The `-modules.yaml` file is the output of the script. It can be deleted. See the new readme in deployments that explains how this all works.